docs/_posts/examples/3400-01-31-data-join.html

```diff
@@ -0,0 +1,70 @@
+--- 
+layout: example 
+category: example 
+title: Join client data to vector tiles 
+description: Use data-driven styles to join client data to vector tiles
```

Comment:
Can you think of a way to frame this title & description that captures the problem being solved as well as the proposed solution?